packages/flutter_tools/lib/src/test/runner.dart

```diff
@@ -89,14 +123,65 @@ Future<int> runTests(
 
   // Call package:test's main method in the appropriate directory.
   final Directory saved = fs.currentDirectory;
+
+  final Function printReloadMessage = () {
```

Comment:
Make this a private top-level method, don't use `Function x = ` declarations.

Reply:
Done